cpu/samd5x/include/periph_cpu.h

```diff
@@ -70,13 +70,21 @@ enum {
  * @name   SAMD5x GCLK definitions
  * @{
  */
-enum {
-    SAM0_GCLK_MAIN = 0,                 /**< 120 MHz main clock       */
-    SAM0_GCLK_32KHZ,                    /**< 32 kHz clock             */
-    SAM0_GCLK_TIMER,                    /**< 4-8 MHz clock for xTimer */
-    SAM0_GCLK_PERIPH,                   /**< 12-48 MHz (DFLL) clock   */
-    SAM0_GCLK_100MHZ,                   /**< 100MHz FDPLL clock       */
-};
+#ifndef SAM0_GCLK_MAIN
+#define SAM0_GCLK_MAIN 0                /**< 120 MHz main clock       */
+#endif
```

Comment:
```suggestion
#define SAM0_GCLK_MAIN 0                /**< 120 MHz main clock       */
```
Main Clock is fixed to GCLK0

Reply:
Is it possible that `SAM0_GCLK_32KHZ` is also fixed? When trying to overwrite it I didn't get any text on the terminal.

Reply:
It shoudln't be, unless it's hard-coded somewhere.
But maybe we should add some `static_assert()` to ensure no two GCLKs are configured to the same ID.